FileShareClient/Models/SearchBatchAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    public class SearchBatchAttribute 
```

Comment:
Please rename `SearchBatchAttribute` to `BatchAttributesSearchAttribute`
```suggestion
    public class BatchAttributesSearchAttribute
```

Reply:
Updated as suggested. Thanks